src/ObservabilityOptions.cs

```diff
@@ -0,0 +1,17 @@
+// ------------------------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.  See License in the project root for license information.
+// ------------------------------------------------------------------------------
+
+using System;
+
+namespace Microsoft.Kiota.Authentication.Azure;
+/// <summary>
+/// Holds the tracing, metrics and logging configuration for the authentication provider
+/// </summary>
+public class ObservabilityOptions {
+    private static readonly Lazy<string> _name = new Lazy<string>(() => typeof(ObservabilityOptions).Namespace);
```

Comment:
Similar to previous comment,
There exists a `ObservabilityOptions` class which only holds the `TracerInstrumentationName` which is essentially the namespace name while this is done differently in https://github.com/microsoft/kiota-abstractions-dotnet/pull/34. 
Is this needed if we only need the name? Or do we foresee more properties added?

Reply:
The reason why I added that is because I suspect other options that are configurable by the developer will show up once we look at implementing metrics and logging. I don't know what however. And I wanted to be able to add those options later without having to ship a breaking change so we don't need to hold the GA of dotnet for the implementation of those aspects.

Reply:
Thanks for clearing that up.